CHANGELOG.md

```diff
@@ -24,6 +24,7 @@
 * [7264](https://github.com/grafana/loki/pull/7264) **bboreham**: Chunks: decode varints directly from byte buffer, for speed.
 * [7263](https://github.com/grafana/loki/pull/7263) **bboreham**: Dependencies: klauspost/compress package to v1.15.11; improves performance.
 * [7270](https://github.com/grafana/loki/pull/7270) **wilfriedroset**: Add support for `username` to redis cache configuration.
+* [7602](https://github.com/grafana/loki/pull/7602) **vmax**: Add decolorize filter to easily parse colored logs.
```

Comment:
could you please move your changelog entries to new sections from https://github.com/grafana/loki/pull/7606

you added your entries to Loki 2.7.0 changelog.

Reply:
@vlad-diachenko rebased & moved to `Unreleased` section in https://github.com/grafana/loki/pull/7602/commits/bb901e4691a015976ab37aa33bdb7adad41cac7d